geotrek/tourism/filters.py

```diff
@@ -32,3 +32,11 @@ class Meta(StructureRelatedFilterSet.Meta):
         model = TouristicEvent
         fields = StructureRelatedFilterSet.Meta.fields + [
             'published', 'type', 'themes', 'after', 'before', 'approved']
+
+
+class TouristicEventApiFilterSet(django_filters.FilterSet):
+    ends_after = django_filters.DateFilter('end_date', lookup_type='gte')
+
+    class Meta:
+        model = TouristicEvent
+        fields = ['end_date', ]
```

Comment:
Obviously you haven't seen my edit.
For the record, i mistakenly commited the test to another branch on my machine so i have to tidy things up before adding to this PR.